Index: src/app/auth/login/page.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import Login from \"../../../../components/views/login/page\";\r\n\r\nconst LoginPage = () => {\r\n    return (\r\n        <>\r\n            <h6>master</h6>\r\n                        <h6>master</h6>\r\n\r\n            <Login />\r\n            <footer className=\"footer\">\r\n                <div className=\"copyright-text\">\r\n                    BookShop ©2023 Created By AR7B.\r\n                </div>\r\n            </footer>\r\n        </>\r\n    );\r\n};\r\n\r\nexport default LoginPage;\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/app/auth/login/page.tsx b/src/app/auth/login/page.tsx
--- a/src/app/auth/login/page.tsx	(revision 3f5828c77577c886cb7d91208d756da9f8cf2e8e)
+++ b/src/app/auth/login/page.tsx	(date 1713597018362)
@@ -4,8 +4,7 @@
     return (
         <>
             <h6>master</h6>
-                        <h6>master</h6>
-
+            <h6>master</h6>
             <Login />
             <footer className="footer">
                 <div className="copyright-text">
